lib/builders/testArgBuilder.js

```diff
@@ -0,0 +1,40 @@
+module.exports = (value) => {
+    let args = [];
+    if(!value){
+        return args;
+    }
+    if (value.settings) {
+        args = args.concat(['--settings', value.settings]);
+    }
+    if (value.listTests) {
+        args = args.concat(['--list-tests']);
+    }
+    if (value.testAdapterPath) {
+        args = args.concat(['--test-adapter-path', value.testAdapterPath]);
+    }
+    if (value.logger) {
+        args = args.concat(['--logger', value.logger]);
+    }
+    if (value.configuration) {
+        args = args.concat(['--configuration', value.configuration]);
+    }
+    if (value.framework) {
+        args = args.concat(['--framework', value.framework]);
+    }
+    if (value.output) {
+        args = args.concat(['--output', value.output]);
+    }
+    if (value.diag) {
```

Comment:
is the name of the command line arg with the same name. The actual variable should be documented for users.

![image](https://cloud.githubusercontent.com/assets/3459800/24726583/ed99b694-1a20-11e7-803b-1e2039ec2472.png)